source/ui/BioXAS/BioXASControlEditor.h

```diff
@@ -63,7 +67,7 @@ public slots:
 	/// Sets whether the control's name is used for the title text.
 	void setUseControlNameAsTitle(bool useName);
 	/// Sets the value.
-	virtual void setValue(double newValue);
+	virtual void setValue(AMNumber newValue);
```

Comment:
Is there a particular reason for using AMNumber instead of double?  If you are going to use AMNumber it should be `const AMNumber &`.